src/components/Sidebar.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { usePostsContext } from '../hooks/usePostsContext';
+import { PostDetails } from './Posts/PostDetails';
```

Comment:
You're importing PostDetails but it's not clear whether its parent directory is at the same level as Sidebar.tsx, you might need to adjust the import path.